crates/turborepo-lib/src/cli.rs

```diff
@@ -280,7 +290,11 @@ pub enum Command {
     Run(Box<RunArgs>),
     /// Unlink the current directory from your Vercel organization and disable
     /// Remote Caching
-    Unlink {},
+    Unlink {
+        /// Specify what should be unlinked (default "remote cache")
+        #[clap(long, value_enum)]
```

Comment:
We can avoid the `Option` here by using `default_value_t`

Reply:
Nice!